### PR TITLE
version 2.0 / with Gene review

### PR DESCRIPTION
**Pull request by gdestuynder**
Several changes/rewording and new section added

### CODE REVIEW

File: Security/Fundamentals/Security_Principles.mediawiki [hunk 1]
```diff
@@ -26,77 +28,87 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
...
+''The process of converting information or data into a code to prevent unauthorized access.''
 
 '''Do'''
-* Encrypt data at rest when not actively used by the system.
-* Use full-disk encryption where available.
-* Encrypt credentials storage databases (Ansible Vault, CredStash, etc.)
-* Encrypt traffic in transit (TLS).
+* Use full-disk encryption where available on systems without physical security (laptops and mobile phones).
+* Encrypt credentials storage databases (Ansible Vault, Credstash, etc.)
+* Encrypt data in transit (during transmission).
```
Comment (mpurzynski) [inline Security/Fundamentals/Security_Principles.mediawiki]: At all times for every system?

File: Security/Fundamentals/Security_Principles.mediawiki [hunk 2]
```diff
@@ -26,77 +28,87 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
...
+* Also encrypt data in transit inside the internal network.
```
Comment (mpurzynski) [inline Security/Fundamentals/Security_Principles.mediawiki]: At all times - for every system? Is a separate Vlan or a physical network with no uplink to other networks a good enough compensating control?

File: Security/Fundamentals/Security_Principles.mediawiki [hunk 3]
```diff
@@ -26,77 +28,87 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
...
 
-''Do not'''
-* Terminate TLS for all services in one location, then transmit the information in clear-text in the rest of the network.
+'''Do not'''
+* Terminate TLS (e.g. with a reverse proxy or load balancer) outside a system and then transmit the data in clear-text across the rest of the network.
 * Use STARTTLS without also disabling clear-text connections.
```
Comment (mpurzynski) [inline Security/Fundamentals/Security_Principles.mediawiki]: do not - redirect users from the http to https, disable the http port (if not necessary)

File: Security/Fundamentals/Security_Principles.mediawiki (hunk 1 above)
Comment (gdestuynder) [inline Security/Fundamentals/Security_Principles.mediawiki]: these are just examples

File: Security/Fundamentals/Security_Principles.mediawiki (hunk 2 above)
Comment (gdestuynder) [inline Security/Fundamentals/Security_Principles.mediawiki]: same :)

File: Security/Fundamentals/Security_Principles.mediawiki (hunk 3 above)
Comment (gdestuynder) [inline Security/Fundamentals/Security_Principles.mediawiki]: you would still need HSTS - discussed irl

File: Security/Fundamentals/Security_Principles.mediawiki (hunk 1 above)
Comment (gene1wood) [inline Security/Fundamentals/Security_Principles.mediawiki]: @mpurzynski Do you have scenarios in mind where encrypting data in transit wouldn't make sense that we should call out here?

Comment (mpurzynski) [inline Security/Fundamentals/Security_Principles.mediawiki]: When you software does not support any kind of encryption and the compensating control is a separate Vlan,. that's not spanned to the core or a dedicated physical network.
One needs to go to a datacenter to plug self to that network.

**Pull request merged**